stdlib/argparse.pyi

```diff
@@ -458,8 +459,43 @@ class _SubParsersAction(Action, Generic[_ArgumentParserT]):
         help: str | None = ...,
         metavar: str | tuple[str, ...] | None = ...,
     ) -> None: ...
-    # TODO: Type keyword args properly.
-    def add_parser(self, name: str, **kwargs: Any) -> _ArgumentParserT: ...
+
+    # The signature of add_parser matches that of ArgumentParser.__init__, except
+    # add_parser returns an ArgumentParser.
+    if sys.version_info >= (3, 9):
+        def add_parser(
+            self,
```

Comment:
You are missing the `name` parameter. The rest of the parameters should be also marked as keyword-only, so that `add_parser("asd1", "asd2")` is an error as it should be.

```suggestion
            self,
            name: str,
            *,
```

Also applies to the pre-3.9 definition.

Reply:
Thanks. I wasn't reading the docs carefully enough yesterday.